src/bgw/job.h

```diff
@@ -13,6 +13,14 @@
 #include "export.h"
 #include "catalog.h"
 
+/* Positive result numbers reserved for success */
+typedef enum JobResult
+{
+	JOB_FAILURE_TO_START = -1,
```

Comment:
I wonder why only `JOB_FAILURE` is not enough here, I guess this is not related to this PR

Reply:
No, it was already there.

Reply:
I don't think the move to this file is necessary.

Reply:
Removed this change.